packages/react/src/views/ChatInput/ChatInput.js

```diff
@@ -422,7 +425,13 @@ const ChatInput = ({ scrollToBottom }) => {
   };
 
   return (
-    <Box className={`ec-chat-input ${classNames}`} style={styleOverrides}>
+    <Box
+      className={`ec-chat-input ${classNames}`}
+      style={styleOverrides}
+      css={css`
+        width: ${showSidebar ? 'calc(100vw - 350px - 3rem)' : '98vw'};
```

Comment:
Can we do anything better than this ? Like any other approach, we can fix this ?

Reply:
@Spiral-Memory , I changed the approach.